src/welcomePage/WelcomePageTilesContainer/tileData.tsx

```diff
@@ -33,6 +35,64 @@ const GitHubIcon = () => <SVG src={github} className={noShrink} />;
 const DiscordIcon = () => <SVG src={discord} className={noShrink} />;
 const ExternalLinkIcon = () => <SVG src={externalLink} className={noShrink} />;
 
+export const CommunityLiveUpdate = ({
+  onTileButtonClick,
+}: {
+  onTileButtonClick: (tileId: string) => void;
+}) => {
+  const [isDismissed, setIsDismissed] = useLocalStorage("community-live-update-dismissed", false);
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <WelcomePageInfoTile
+      id="community-live-update"
+      __backgroundColor="#F5F3FF"
+      __borderColor="#9767E4"
+      position="relative"
+      header={<FormattedMessage defaultMessage="✨ Join Live Community Update" id="dcFARy" />}
+      content={
+        <>
+          <Paragraph>
+            <FormattedMessage
+              defaultMessage="Tune in for live updates with product news, tips, and how teams are building with Saleor."
+              id="FcC842"
+            />
+          </Paragraph>
+          <Paragraph fontWeight="bold" paddingTop={1}>
+            <FormattedMessage defaultMessage="April 24th at 3 PM CET | 9 AM EDT" id="Vrch8P" />
```

Comment:
suggestion: maybe we could use `<FormattedDate>` and `<FormattedTime>` from react-intl to get local time zone of user?

https://formatjs.github.io/docs/react-intl/components#formattedtime